index.bs

```diff
@@ -654,7 +654,6 @@ url: https://w3c.github.io/permissions/#permissions-task-source; type: dfn;
 			1. Set |clipboardItemObject|'s [=/clipboard item=] to |clipboardItem|.
 
 		The <dfn constructor for="ClipboardItem" lt="ClipboardItem(items, options)"><code>new ClipboardItem(<var>items</var>, <var>options</var>)</code></dfn> constructor steps are:
-			1. If |items| is empty, then throw a {{TypeError}}.
```

Comment:
What is the behavior on other browsers? Are they compliant with the spec after this update?

Reply:
@EdgarChen @whsieh